src/app/components/layout/ConfirmProceedDialog.js

```diff
@@ -78,7 +78,9 @@ const ConfirmProceedDialog = ({
             label={cancelLabel || <FormattedGlobalMessage messageKey="cancel" />}
           /> : null }
         <ButtonMain
-          id="confirm-dialog__confirm-action-button"
+          buttonProps={{
```

Comment:
This was the main reason for the test failures. I incorrectly was setting the id, which was not getting to the dom as originally coded